### PR TITLE
remove guidestar from common, change calstatus to cal_step

### PR DESCRIPTION
**Pull request by perrygreenfield**
reverting to JWST name for status of calibration processing ("cal_step"), also removing guidestar from the common schema.

### CODE REVIEW

File: src/rad/resources/schemas/reference_files/flat-1.0.0.yaml [hunk 1]
```diff
@@ -10,37 +10,17 @@ properties:
...
-    properties:
-      datatype:
-        enum: [float32]
-      shape:
-        type: array
-        minItems: 2
-        maxItems: 2
+    datatype: float32
+    ndim: 2
     required: [datatype, shape]
```
Comment (eslavich) [inline src/rad/resources/schemas/reference_files/flat-1.0.0.yaml:15]: We don't need to require these, the tag validations will take care of whatever is necessary

File: src/rad/resources/schemas/wfi_image-1.0.0.yaml [hunk 2]
```diff
@@ -13,81 +13,33 @@ properties:
...
-    properties:
-      datatype:
-        enum: [float32]
-      shape:
-        type: array
-        minItems: 2
-        maxItems: 2
+    datatype: float32
+    ndim: 2
     required: [datatype, shape]
```
Comment (eslavich) [inline src/rad/resources/schemas/wfi_image-1.0.0.yaml:26]: Another unnecessary required

File: src/rad/resources/schemas/common-1.0.0.yaml [hunk 3]
```diff
@@ -47,8 +47,6 @@ properties:
     tag: "tag:stsci.edu:datamodels/roman/velocity_aberration-1.0.0"
   wcsinfo:
     tag: "tag:stsci.edu:datamodels/roman/wcsinfo-1.0.0"
-  guidestar:
-    tag: "tag:stsci.edu:datamodels/roman/guidestar-1.0.0"
-  calstatus:
-    tag: "tag:stsci.edu:datamodels/roman/calstatus-1.0.0"
+  cal_step:
+    tag: "tag:stsci.edu:datamodels/roman/cal_step-1.0.0"
 ...
```
Comment (eslavich) [inline src/rad/resources/schemas/common-1.0.0.yaml:52]: Our code style checks generally demand a newline at the end of a file

File: src/rad/resources/schemas/wfi_image-1.0.0.yaml (hunk 2 above)
Comment (perrygreenfield) [reply src/rad/resources/schemas/wfi_image-1.0.0.yaml:26]: escaped my substitutions. Sneaky bugger.

File: src/rad/resources/schemas/reference_files/flat-1.0.0.yaml (hunk 1 above)
Comment (perrygreenfield) [reply src/rad/resources/schemas/reference_files/flat-1.0.0.yaml:15]: Another escapee.

File: src/rad/resources/schemas/common-1.0.0.yaml (hunk 3 above)
Comment (perrygreenfield) [reply src/rad/resources/schemas/common-1.0.0.yaml:52]: done.

**Review by eslavich — APPROVED** [review]
LGTM!

**Pull request merged**